src/restLink.ts

```diff
@@ -326,6 +326,25 @@ export const validateRequestMethodForOperationType = (
   }
 };
 
+//Used for any Error for data from the server
```

Comment:
If you change this into `/** …` comments it will get picked up as documentation on the type!